### ISSUE TITLE
1.3.1 Tables

### ISSUE THREAD

**Issue by svetlanaangelova** [samheather/iapt2group1#95]
- on Populate project page both tables need a caption and a summary
- on Populate project page both tables need a header for the last column
- on Dashboard page table needs a caption and a summary
- Same on Transcriptions page
- on Summary of Transcriptions tables is missing a caption and a better summary
